### PR TITLE
Make sure all the existing code follows the coding style

### PR DESCRIPTION
**Pull request by TheSecondReal0**

### CODE REVIEW

**Comment by nicemicro** [comment]
could you please check if all these guidelines are in the coding style guide md file and add them there if not?
maybe we should also give some numbers to each point in the style guide for easier reference

File: code-styleguide.md [hunk 1]
```diff
@@ -1,40 +1,151 @@
...
+# 1. Naming conventions
+## 1.1. Descriptive names
+The function / variable / class names should hint at their role.
 ### Bad
+````
     func computerTime(d):
+````
 ### Good
+````
     func computeTime(daysElapsed):
```
Comment (moxvallix) [inline code-styleguide.md]: should this be computerTime to match the bad example above?

File: code-styleguide.md [hunk 2]
```diff
@@ -1,40 +1,151 @@
...
-## Be consistent
+````
+## 1.2. Consistency
+Look at how others named things in the code, and foolow that.
```
Comment (moxvallix) [inline code-styleguide.md]: follow, not foolow

**Review by moxvallix — CHANGES_REQUESTED** [review]
i think this file should be renamed either to STYLEGUIDE.md, CODE_STYLE.md, or code_styleguide.md to meet our naming schemes. The all caps file name convention seems to be for the more meta files in base, so maybe STYLEGUIDE is a fit?

**Review by nicemicro — APPROVED** [review]
I like it.

**Review by moxvallix — APPROVED** [review]
looks good

**Pull request merged**